prompting/conversation.py

```diff
@@ -42,7 +42,10 @@ def create_task(
     if dataset is None:
         raise ValueError(f"Dataset {dataset_name} not found")
     else:
-        dataset = dataset()
+        if task_name == SummarizationTask.name:
+            dataset = dataset(selector='all')
```

Comment:
```suggestion
            dataset = dataset(selector="all")
```
Should we just modify the default argument of `WikiDataset.get` for `selector="all"`, to make the code less coupled?